app/assets/javascripts/presentations.js.coffee

```diff
@@ -0,0 +1,11 @@
+jQuery ($) ->
+  return if $('.presentations.show').size() == 0
```

Comment:
[The .size() method is functionally equivalent to the .length property; however, the .length property is preferred because it does not have the overhead of a function call.](https://api.jquery.com/size/)
らしいです